extensions/Vendor/BLENDER_physics/README.md

```diff
@@ -0,0 +1,182 @@
+# BLENDER\_physics
+
+## Contributors
+
+* Mitchell Stokes, Blender, <mailto:mogurijin@gmail.com>
+* Daniel Stokes, Blender, <mailto:kupomail@gmail.com>
+* Steven Vergenz, AltspaceVR, <mailto:steven@altvr.com>
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against glTF 1.0 and 2.0 specs.
+The only difference for this extension for 1.0 vs 2.0 is the `mesh-id`.
+
+## Overview
+
+This extension extends nodes to include information necessary to construct rigid body objects for a physics engine (e.g., [Bullet](https://bulletphysics.org/)).
+
+## glTF Schema Updates
+
+A `BLENDER_physics` object is added to the extensions list of any `node` that should participate in the physics simulation.
+The properties available are listed in the table below.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**collisionShape**|`string`|The shape a physics simulation should use to represent the node|No, default: `BOX`|
+|**mass**|`number`|The 'weight', irrespective of gravity, of the node|No, default: `1.0`|
+|**static**|`boolean`|Whether or not the Node should be moved by physics simulations|No, default: `false`|
+|**boundingBox**|`array`|The dimensions of the local (i.e., does not include the node's transform) bounding box of the collision shape centered on the origin|Yes|
+|**primaryAxis**|`number`|The axis to use for the height of the collision shape|No, default: `1`|
+|**mesh**|`glTF id`|The ID of the mesh to use for `CONVEX_HULL` and `MESH` collision shapes|No, default: `node's mesh` if it exists, otherwise use `BOX` shape|
+|**offsetMatrix**|`array`|A 4x4 transform matrix applied to the collision shape in addition to the node's transform|No, default: `[ 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0 ]`|
+|**offsetRotation**|`array`|A rotation offset (as a quaternion) applied to the physics shape in addition to the node's rotation|No, default: `[0.0, 0.0, 0.0, 1.0]`|
+|**offsetScale**|`array`|A non-uniform scale offset applied to the collision shape in addition to the node's scale|No, default: `[1.0, 1.0, 1.0]`|
+|**offsetTranslation**|`array`|A translation offset applied to the collision shape in addition to the node's translation|No, default: `[0.0, 0.0, 0.0]`|
+|**collisionGroups**|`integer`|A 32-bit bit field representing the node's group membership|No, default: `1`|
+|**collisionMasks**|`integer`|A 32-bit bit field representing what groups the node can collide with|No, default: `1`|
+
+**Collision Shapes**
+
+Below are the allowed values for `collisionShape`.
+When consuming this extension, an application should construct the `collisionShape` with parameters that best fill the `boundingBox`.
+The shape offset should be applied to the shape prior to the node's transform.
+
+* `BOX`
+* `SPHERE`
+* `CAPSULE`
```

Comment:
Perhaps note that the capsule's height must be equal or greater to the diameter.